SDK/CalendarStoreViewController.swift

```diff
@@ -63,9 +71,9 @@ public final class CalendarStoreViewController: UITabBarController {
      - parameter pageIdentifier: The page identifier for the the home page.
      - parameter title: The title for the `navigtaion bar` in the home page.
      */
-    public init(apiClient: Api, pageIdentifier: String?, title: String?) {
+    public init(pageIdentifier: String?, title: String?) {
         // Initialization
-        self.apiClient = apiClient
+        self.apiClient = SchedJoulesApi(accessToken: apiKey)
```

Comment:
Not a fan of this change. We expect partners to keep their API key secret. If they pass it to the constructor they can decide themselves where and how to store it. If they choose to store it in the `Info.plist` from where any other SDK they use can read it it's up to them, but we shouldn't make it mandatory.
That's also not how other services do it, see how you have to provide the APK Key to Google Services SDK: https://developers.google.com/maps/documentation/ios-sdk/get-api-key